stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessage.kt

```diff
@@ -37,43 +41,74 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  * @param message Message to show.
  * @param modifier Modifier for styling.
  * @param onLongItemClick Handler when the item is long clicked.
+ * @param onQuotedMessageClick Handler for quoted message click action.
  */
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 public fun QuotedMessage(
     message: Message,
     modifier: Modifier = Modifier,
     onLongItemClick: (Message) -> Unit,
+    onQuotedMessageClick: (Message) -> Unit,
```

Comment:
This is another breaking change - I would suggest making a duplicate of the quoted message that doesn't have this param and keeps its behavior as-is.

Then we can apply a deprecation notice on it and have this new function be the standard.